mktrayicon.c

```diff
@@ -220,22 +216,36 @@ int main(int argc, char **argv)
 	XInitThreads(); /* see http://stackoverflow.com/a/18690540/472927 */
 	gtk_init(&argc, &argv);
 
-	if (argc == 4 && strcmp(argv[1], "-i") == 0) {
-		start_icon = argv[2];
-	}
-
 	if (argc == 1) {
-		printf("Usage: %s [-i ICON] FIFO\n", *argv);
-		printf("Listen to FIFO for system tray icon specifications\n");
+		printf("Usage: %s [-i ICON] [-t TOOLTIP] [FIFO]\n", *argv);
```

Comment:
This is now a little weird, because we actually _require_ `-i` if you provide `-t`. Maybe we should add a real argument parsing lib :p

Reply:
True. 
I don't have time for it today, will get around to it when I can. I'll be going away on vacation with family this weekend, so it probably won't be until late next week.